lib/engines/content_object_store/app/models/content_object_store/content_block_version.rb

```diff
@@ -0,0 +1,5 @@
+class ContentObjectStore::ContentBlockVersion < ApplicationRecord
```

Comment:
This is a good idea - in my head we'd be leaning on the Editions to get a picture of the history, but would be better to spell it out